app/src/main/res/layout/activity_sign_up.xml

```diff
@@ -0,0 +1,242 @@
+        android:layout_marginStart="@dimen/margin_text_valid_email"
+        android:layout_marginEnd="@dimen/margin_text_valid_email"
+        android:textColor="@color/red"
+        android:text=""
+        android:textSize="@dimen/hint_text_size_email"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/edit_text_email" />
+
+    <TextView
+        android:id="@+id/text_view_password"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/margin_text_view_password"
+        android:layout_marginTop="@dimen/margin_top_text_view_password"
+        android:labelFor="@+id/edit_text_password"
+        android:text="@string/password"
+        android:textColor="@color/white"
+        android:textSize="@dimen/text_size_text_view_password"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/text_valid_email" />
+
+    <EditText
+        android:id="@+id/edit_text_password"
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/height_text_password"
+        android:layout_marginStart="@dimen/margin_text_password"
+        android:layout_marginEnd="@dimen/margin_text_password"
+        android:minEms="8"
+        android:maxEms="20"
+        android:text=""
+        android:textColor="@color/white"
+        android:inputType="textPassword"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/text_view_password" />
+
+    <TextView
+        android:id="@+id/text_valid_password"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/margin_text_valid_password"
+        android:layout_marginEnd="@dimen/margin_text_valid_password"
+        android:textColor="@color/red"
+        android:text=""
+        android:textSize="@dimen/hint_text_size_valid_password"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/edit_text_password" />
+
+    <androidx.appcompat.widget.AppCompatCheckBox
+        android:id="@+id/remember_check_box"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/margin_remember_check_box"
+        android:layout_marginTop="@dimen/margin_top_remember_check_box"
+        android:checked="true"
+        android:text="@string/remember_me"
+        android:textColor="@color/white"
+        android:textSize="@dimen/text_size_remember_check_box"
+        app:buttonTint="@color/white"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/text_valid_password" />
+
```

Comment:
if the margins are the same, you can use margin_horizontal or margin_vertical

Reply:
It was fixed